serializers/web/invoice.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "countries"
```

Comment:
Why do we need this?

Reply:
We use country codes from it in this serializer. https://github.com/ubicloud/ubicloud/blob/4408a5184298c3f215849a75c8a9fa2d43c38e47/serializers/web/invoice.rb#L43

The reason it worked before is that it was imported at the billing route, and the invoice serializers were only used there.
https://github.com/ubicloud/ubicloud/blob/4408a5184298c3f215849a75c8a9fa2d43c38e47/routes/web/project/billing.rb#L4